components/SwipeListView.js

```diff
@@ -27,6 +27,10 @@ class SwipeListView extends Component {
         }
     }
 
+    componentDidUpdate(nextProps) {
+        if (nextProps.refreshing) this.safeCloseOpenRow();
+    }
```

Comment:
Ah sorry @PAronofsky actually I think another change is required. `componentDidUpdate`'s argument is `prevProps`, not `nextProps`.

And since it is `prevProps` you may just want to use `if (this.props.refreshing) `

Reply:
Ah sorry! My bad. It seemed to work ok. We've been using hooks/function components so I'm a little rusty on the component lifecycle methods. Will submit a change in a bit.

Reply:
Should be ok, thanks for your time, @jemise111 !

Reply:
Thanks, looks good!